OpenRA.Mods.Common/Traits/Render/WithDamageOverlay.cs

```diff
@@ -29,6 +29,13 @@ public class WithDamageOverlayInfo : ITraitInfo, Requires<RenderSpritesInfo>
 		[SequenceReference("Image")]
 		public readonly string EndSequence = "end";
 
+		[PaletteReference("IsPlayerPalette")]
+		[Desc("Custom palette name")]
```

Comment:
For consistency we should have a `. ` in the Desc.

Reply:
I copy-pasted those from some other trait, but I guess you're right in the context of this trait.